modules/curl/8.4.0.bcr.1/patches/add_build_file.patch

```diff
@@ -0,0 +1,265 @@
+diff --git a/BUILD.bazel b/BUILD.bazel
+new file mode 100644
+index 000000000..d11e051cd
+--- /dev/null
++++ b/BUILD.bazel
+@@ -0,0 +1,259 @@
++# Copyright The OpenTelemetry Authors
++# SPDX-License-Identifier: Apache-2.0
++
++load("@bazel_skylib//rules:common_settings.bzl", "bool_flag")
++
++package(features = ["no_copts_tokenization"])
++
++bool_flag(
++    name = "use_mbedtls",
++    build_setting_default = False,
++)
++
++config_setting(
++    name = "use_mbedtls_setting",
++    flag_values = {":use_mbedtls": "true"},
++)
++
++bool_flag(
++    name = "http_only",
++    build_setting_default = True,
++)
++
++config_setting(
++    name = "http_only_setting",
++    flag_values = {":http_only": "true"},
++)
```

Comment:
Could you explicitly mark this and the one further up as private? I'm not sure what the status on implicit private visibility for config_settings is in older versions of Bazel.

Reply:
done